CollapseLauncher/Classes/InstallManagement/BaseClass/InstallManagerBase.cs

```diff
@@ -896,8 +1073,15 @@ public virtual async ValueTask ApplyHdiffListPatch()
 
                         await Task.Run(() =>
                         {
-                            patcher.Initialize(patchPath);
-                            patcher.Patch(sourceBasePath, destPath, true, _token.Token, false, true);
+#if DEBUG
+                            try
+                            {
+#endif
+                                patcher.Initialize(patchPath);
+                                patcher.Patch(sourceBasePath, destPath, true, _token.Token, false, true);
+#if DEBUG
+                            } catch { }
```

Comment:
Is there a reason why the empty catch clause only available in debug?

Reply:
Sometimes it's annoying to have exception notification in VS, especially with an exception caused by the already patched file. Will be removed later.

Reply:
Already removed